libmach/test.lisp

```diff
@@ -0,0 +1,64 @@
+;; Tests the behavior of `libmach` using Common Lisp's CFFI
```

Comment:
Just curious @zack466 which language do you actually plan to use this from? I've personally never used lisp, but would be interested in maintaining some official Go bindings potentially.

Reply:
Common Lisp, haha

CFFI code is dead simple in Lisp (as you can see in `test.lisp`), which is really nice. I was actually originally trying to work with bgfx, but it was a huge pain to compile and use properly, so I eventually gave up. With Mach, on the other hand, a simple `zig build` does the trick!